test/extended/util/image_helpers.go

```diff
@@ -97,3 +97,15 @@ func CreateResource(jsonFilePath string, oc *CLI) error {
 	err := oc.Run("create").Args("-f", jsonFilePath).Execute()
 	return err
 }
+
+//NewApp performs an oc new-app invocation from the supplied json file
+func NewApp(jsonFilePath string, oc *CLI) error {
+	err := oc.Run("new-app").Args(jsonFilePath).Execute()
+	return err
```

Comment:
is this worth to have special function? I think reading the code will tell you what this does.... having NewApp() makes you go find the definition if something explode :)